web/src/components/molecules/Request/Details/Editor.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from "@emotion/styled";
+import { useCallback, useState } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import { useT } from "@reearth-cms/i18n";
+
+const { TextArea } = Input;
+
+type Props = {
+  onCommentCreate: (content: string) => Promise<void>;
+};
+
+const RequestEditor: React.FC<Props> = ({ onCommentCreate }) => {
+  const [submitting, setSubmitting] = useState(false);
+  const [form] = Form.useForm();
+  const t = useT();
+
+  const handleSubmit = useCallback(async () => {
+    try {
+      setSubmitting(true);
+      const values = await form.validateFields();
+      await onCommentCreate?.(values.content);
+      form.resetFields();
+      setSubmitting(false);
+    } catch (info) {
+      setSubmitting(false);
```

Comment:
You don't need these two setSubmitting(false) because you handle it in the `finally`